common/changes/leddie24-fix-name-prop-ChoiceGroup-CheckBox_2017-01-13-22-13.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "ChoiceGroup and CheckBox: Fix `name` prop not being set properly on radio and input elements",
```

Comment:
Maybe something like "Added props to set a custom 'name' attribute on rendered elements." Want to make sure that people understand a new prop has been added, and that we haven't fixed/changed anything existing.